app/services/idv/session.rb

```diff
@@ -70,8 +70,12 @@ def clear
       user_session.delete(:idv)
     end
 
+    def phone_confirmation_complete?
+      phone_confirmation == true && params['phone_confirmed_at'].present?
```

Comment:
The problematic `phone_confirmation` variable still exists. I'd like to put up another commit that refactors this a bit to rename `phone_confirmation` to `vendor_phone_confirmation` and adds `user_phone_confirmation`.